crypto/pkcs11key/pkcs11key.go

```diff
@@ -34,22 +34,24 @@ type PKCS11Key struct {
 	// The PKCS#11 library to use
 	module *pkcs11.Ctx
 
-	// The name of the slot to be used.
-	// We will automatically search for this in the slot list.
-	slotDescription string
-
 	// The PIN to be used to log in to the device
 	pin string
 
+	// The name of the slot to be used, or "" to use any slot
+	slotDescription string
+
+	// The label of the token to be used, or "" to use any token
```

Comment:
I'm fine with slotDescription being optional, but I think at least tokenLabel should be mandatory. Otherwise it's easy to accidentally leave out the token label, attempt to log into the wrong token, fail repeatedly, and lock that incorrect token.